lighthouse-core/report/html/renderer/util.js

```diff
@@ -596,6 +596,8 @@ Util.UIStrings = {
   runtimeSettingsUANetwork: 'User agent (network)',
   /** Label for a row in a table that shows the estimated CPU power of the machine running Lighthouse. Example row values: 532, 1492, 783. */
   runtimeSettingsBenchmark: 'CPU/Memory Power',
+  /** Label for a row in a table that shows the version of the Axe library used. Example row values: 2.1.0, 3.2.3 */
+  runtimeSettingsAxeVersion: 'Axe Version',
```

Comment:
```suggestion
  runtimeSettingsAxeVersion: 'Axe version',
```

(title case is ugly :)

Reply:
it's possible "Axe" will end up translated in this string. The official way to deal with that is use ``'`Axe` version'`` here, but we don't do anything with markdown in the runtime settings currently, so if we do want to make sure it won't be translated, either we'd need to add markdown parsing there or live with the backticks.

Reply:
hard 👎 on `Axe version` (non-title case is ugly :P)

As for the substance though...

> or just live with it possibly translated in some languages

This seems far more consistent with our handling of other library and product names. I couldn't find any instance of us backticking `Lighthouse` in our strings for example.

Reply:
> hard 👎 on `Axe version` (non-title case is ugly :P)

say hello to `Network throttling`, `CPU throttling`, `User agent (host)`, etc above this :P:P

> This seems far more consistent with our handling of other library and product names. I couldn't find any instance of us backticking `Lighthouse` in our strings for example.

Yeah, more of an FYI. Interestingly there are 17 `Lighthouse`s in `en-US.json`, 16 in most of the other language files I tested, but only 14 in `ru.json` 😰

Reply:
> title case 

(for the record, this isn't at all important right now, but we should probably get our act together on consistency in these entries at some point)

Reply:
Agreed we should be consistent eventually, just didn't want another vote in that section for non-title case ;)